packages/airnode-node/src/coordinator/calls/disaggregation.ts

```diff
@@ -1,63 +1,59 @@
 import flatMap from 'lodash/flatMap';
-import { logger, PendingLog } from '@api3/airnode-utilities';
+import { logger } from '@api3/airnode-utilities';
 import {
   AggregatedApiCallsById,
   ApiCall,
   Request,
   CoordinatorState,
   EVMProviderState,
-  GroupedRequests,
   LogsData,
   ProviderState,
-  RequestErrorMessage,
   RequestStatus,
+  UpdatedRequests,
 } from '../../types';
 
-export interface RequestsWithLogs {
-  readonly logs: PendingLog[];
-  readonly requests: GroupedRequests;
-}
-
 function updateApiCallResponse(
```

Comment:
Rename to `updateApiCallResponses`? (now that it receives multiple apiCalls)

Reply:
Done! Thanks.